packages/suite/src/components/wallet/AccountMode/BackendDisconnected.tsx

```diff
@@ -60,4 +58,25 @@ const Disconnected = () => {
     );
 };
 
+const Disconnected = () => {
+    const { blockchain, selectedAccount, online } = useSelector(state => ({
+        blockchain: state.wallet.blockchain,
+        selectedAccount: state.wallet.selectedAccount,
+        online: state.suite.online,
+    }));
+
+    if (!online) return null;
```

Comment:
So in case that is not online we do not display the `<BackendDisconnected />`?
Because we already show the offline message?

Reply:
Exactly. It should be equivalent to the previous code:
```
if (!blockchain.connected && state.suite.online) {
    mode.push('backend-disconnected');
}
```